pallets/aggregate/src/lib.rs

```diff
@@ -494,9 +602,21 @@ pub mod pallet {
 
     /// Domains storage
     #[pallet::storage]
-    pub(crate) type Domains<T: Config> =
+    pub type Domains<T: Config> =
         StorageMap<Hasher = Blake2_128Concat, Key = u32, Value = Domain<T>>;
 
+    // Storage for held amounts per domain and account to pay dispatch fees
```

Comment:
Same comment as above

Reply:
does not apply anymore as we don't need this storage variable anymore